__tests__/client/prerender.spec.js

```diff
@@ -161,7 +160,7 @@ describe('moveHelmetScripts', () => {
       { src: 'baz.js', helmet: true },
     ]
       .map(createScript)
-      .forEach((script) => document.body.appendChild(script));
+      .forEach((script) => document.body.append(script));
```

Comment:
Was this preferred by the linter?

Reply:
yes https://github.com/sindresorhus/eslint-plugin-unicorn/blob/main/docs/rules/prefer-dom-node-append.md